formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -319,3 +319,28 @@ let myFun
    */
 /* Bucklescript FFI item attributes */
 external imul : int => int => int = "Math.imul" [@@bs.val];
+
+class type _x = {
+                  method height: int
```

Comment:
@jordwalke This bug is introduced when `[@bs]` attribute is [appended](https://github.com/facebook/reason/pull/716/files#diff-7dce44be99776d19af8c2d7b0d14e2c7R4737). It makes braces to be aligned on one column. 

I know nothing about this codebase, but after experimenting a little bit, it seems like the problem is somewhere else. I think that brace indentation logic is causing it but cannot figure out where it happens. Any tips appreciated.